README.md

```diff
@@ -3,7 +3,7 @@ shopify-sinatra-app [![Build Status](https://travis-ci.org/kevinhughes27/shopify
 
 "A classy shopify app"
 
-shopify-sinatra-app is lightweight extension for building Shopify apps using Sinatra. It comes with the Shopify API gem for interacting with the Shopify API and uses the Shopify omniauth gem to handle authentication. The framework itself provides a handful of helper methods to make creating your app as easy as possible. The framework was designed with deployment to Heroku in mind and following the instructions below I've been able to create a new application from scratch, deploy it to Heroku and install on my live test shop in less than 5 minutes.
+shopify-sinatra-app is lightweight extension for building Shopify apps using Sinatra. It comes with the Shopify API gem for interacting with the Shopify API and uses the Shopify omniauth gem to handle authentication (it does not support Private App token based auth). The framework itself provides a handful of helper methods to make creating your app as easy as possible. The framework was designed with deployment to Heroku in mind and following the instructions below I've been able to create a new application from scratch, deploy it to Heroku and install on my live test shop in less than 5 minutes.
 
```

Comment:
I definitely understand where you are coming from but in my opinion your situation (private app) is an edge case and I think adding specific documentation for it might confuse the average user more than it would help.

The main purpose of this project and https://github.com/Shopify/shopify_app is to help with the complexity of having your app installed on multiple shops and doing the oauth etc. Private apps are easy since you just have a password and you can start coding / scripting for your shop. None of the code that determines which shop and shopify token to use is required which is 90% of what these projects are for. Does that make sense?